test_team_2.py

```diff
@@ -21,6 +21,24 @@ def test_url(self):
 
     def test_uk_nhs(self):
         """Test uk_nhs functionality"""
+        # positive testcases
+        prefix = ['123', '245', '532']
+        mid = ['333', '444', '555']
+        suffix = ['6789', '2232', '4832']
+
+        # Generates a testcase by utting the prefix, mid, and suffix together
+        test_cases = [f"My NHS number is {p}-{m}-{s}" for p in prefix for m in mid for s in suffix]
```

Comment:
I think you need to fix the implementation of using "anoymize_text" in your test cases. This seems to be the reason your code is not passing the pytest and getting an AssertionError.

Reply:
I have resolved the issue all test shoudl pass now